netlify.toml

```diff
@@ -0,0 +1,3 @@
+[build]
+	publish = "build/singlehtml"
+	command = "make singlehtml"
```

Comment:
can we do the latexpdf as well?

Reply:
I guess we can't do it via netlify then. This options only define what we want to expose via our webservices. If we want to make it downloadable - then we need to trigger that via the CI.

I can take care of that once this is done.